src/qibocal/protocols/characterization/utils.py

```diff
@@ -165,10 +182,23 @@ def spectroscopy_plot(data, qubit, fit: Results = None):
         row=1,
         col=1,
     )
+    fig.add_trace(
+        go.Scatter(
+            x=np.concatenate((frequencies, frequencies[::-1])),
```

Comment:
```suggestion
            x=np.concatenate((frequencies, frequencies[::-1]))*HZ_TO_GHZ,
```
In the report we should show `GHz` and not `Hz` to avoid having a factor 10^9 on the x-axis.